content/posts/sentakki/index.md

```diff
@@ -17,7 +17,7 @@ summary: "A custom ruleset based on Sega&rsquo;s arcade rhythm game maimai"
 - Availability : Available publicly
 - GitHub Repositories : [LumpBloom7/sentakki](https://github.com/LumpBloom7/sentakki)
 - Discord : [hishikki Dev Server](https://discord.gg/CQPNADu)
-- Support : [Ko-fi](https://ko-fi.com/bloom) or [PayPal](https://www.paypal.com/paypalme/DerrickTimmermans)
+- Support : [Ko-fi](https://ko-fi.com/bloom), [PayPal](https://www.paypal.com/paypalme/DerrickTimmermans) or [GitHub Sponsor](https://github.com/sponsors/LumpBloom7)
```

Comment:
Minor nit, but I'd want GitHub sponsors to be ahead of the other options, since that would be my preferred method.

Reply:
OK, I will rearrage it.